src/components/Board/board.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Row from "../Row/row";
+import styled from 'styled-components';
+
+type BoardData = {
+  boardData: Array<Array<number>>
+}
+
+const StyledTable = styled.table`
+  border-collapse: collapse;
+  border: 1px solid black;
+  margin-right: 0.5em;
+  margin-top: 1em;
+`
+
+const Board = ({ boardData }: BoardData) => {
+  return (
+    <StyledTable data-testid="board_table">
+      <tbody>
+        {boardData.map((row, rowNumber) => <Row key={rowNumber} rowData={row} />  )}
```

Comment:
(copied from https://github.com/dixons/js-academy/pull/13)
For this usecase it is probably ok. But generaly, using array index as `key` is not good practice and can lead to buggy behaviour.

More resources: 
- https://reactjs.org/docs/lists-and-keys.html#keys
- https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Good point, I will study more about the key prop.